### PR TITLE
Training add more pages

### PR DESCRIPTION
**Pull request by klieret**

### CODE REVIEW

File: _training/curriculum.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: "Towards a HEP Software Training curriculum"
+layout: plain
+---
+
+<style type="text/css">
```
Comment (klieret) [inline _training/curriculum.md:6]: Probably not the best style to just paste this here, but seems to work well.

**Comment by klieret** [comment]
Hi @graeme-a-stewart . Thanks your feedback on #752. I tested this setup locally and it seems to work well. What do you think?

File: _training/curriculum.md (hunk 1 above)
Comment (graeme-a-stewart) [reply _training/curriculum.md:6]: I don't think it's so important!

**Comment by graeme-a-stewart** [comment]
Yes, I think it looks good - content and technical solution.

Shall I merge it now?

**Comment by klieret** [comment]
Thanks @graeme-a-stewart

**Pull request merged**